shared.bazelrc

```diff
@@ -20,6 +20,9 @@ common --module_mirrors=https://bcr.cloudflaremirrors.com
 # Avoid being broken by version requirement changes in transitive deps.
 common --check_direct_dependencies=error
 
+# Enable proto toolchain resolution to use pre-built protoc.
+common --incompatible_enable_proto_toolchain_resolution
```

Comment:
https://github.com/protocolbuffers/protobuf/commit/4986a7722460e3163f37c98da1cb47f52c6406e1 mad be needed to avoid further ways in which Protobuf may be built from source. We could pin to that commit now, but can also just see that after the next release.